### PR TITLE
Mc ab strming

### PR DESCRIPTION
**Pull request by FBRTMaka**
@birdage , @DanielJMaher , updated to reflect changes for streaming data and param filtering.
Todo
- [x] add TOC streaming container connection, allowing the user to deselect after filtering
- [x] add remove icon for container on streaming data page

### CODE REVIEW

**Comment by FBRTMaka** [comment]
@DanielJMaher , @Bobfrat , @oceanzus , @ednad , You'll need to reinstall the latest highcharts to get the download buttons working

File: ooiui/static/js/views/common/MapView.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 var MapView = Backbone.View.extend({
-    gliderCollection:null,
-    initialize: function() {
-        var self = this;
+    derender: function() {
+        this.map = '';
```
Comment (birdage) [inline ooiui/static/js/views/common/MapView.js]: map as a string? not null?

File: ooiui/static/js/views/common/TOCView.js [hunk 2]
```diff
@@ -332,15 +320,16 @@ var AssetItemView = Backbone.View.extend({
         "use strict";
         // If the asset class is an AssetRecord, give the view an ID of the
         // first 8 characters of the Reference Designator
-        var assName, platformName, platformId, label, instrumentId, instrumentName, assId;
+        var assName, platformName, platformId, label, instrumentId, instrumentName, assId, refDes;
```
Comment (birdage) [inline ooiui/static/js/views/common/TOCView.js]: change to assetName?

Comment (FBRTMaka) [inline ooiui/static/js/views/common/TOCView.js]: Ha! you found the easter egg!

**Comment by DanielJMaher** [comment]
@FBRTMaka @birdage If this is getting in for 430 it needs to be merged soon... Any is in a meeting from now until 4.  im thinking the demo has to slide.

**Pull request closed without merging**